src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -200,7 +209,11 @@ Status ServiceBasedActorInfoAccessor::AsyncSubscribe(
 Status ServiceBasedActorInfoAccessor::AsyncUnsubscribe(const ActorID &actor_id,
                                                        const StatusCallback &done) {
   RAY_LOG(DEBUG) << "Cancelling subscription to an actor, actor id = " << actor_id;
-  auto status = actor_sub_executor_.AsyncUnsubscribe(subscribe_id_, actor_id, done);
+  auto status =
+      client_impl_->GetGcsPubSub().Unsubscribe(ACTOR_CHANNEL, actor_id.Binary());
```

Comment:
Is this a synchronous operation? Why not invoke the callback of `done` after the `Unsubscribe` is finished synchronous?

Reply:
Unsubscribe not support callback now